src/lib.rs

```diff
@@ -770,41 +804,46 @@ mod test {
   }
 
   #[test]
-  fn check_large_file_size_over_11_chunks() {
-    // has been tested for 50 chunks
+  fn check_10_and_a_bit_max_chunks() {
```

Comment:
Hi ben just send a commit to Dave to have it back in ok
On 25 Mar 2015 19:44, "Benjamin Bollen" <notifications@github.com> wrote:

> In src/lib.rs
> <https://github.com/dirvine/self_encryption/pull/17#discussion_r27158102>:
>
> > @@ -770,41 +804,46 @@ mod test {
> >    }
> >
> >    #[test]
> > -  fn check_large_file_size_over_11_chunks() {
> > -    // has been tested for 50 chunks
> > +  fn check_10_and_a_bit_max_chunks() {
>
> How did it get to overwrite a function?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dirvine/self_encryption/pull/17/files#r27158102>.
>